pkg/ebpf/controlplane/processes.go

```diff
@@ -95,9 +95,9 @@ func (ctrl *Controller) procTreeForkProcessor(args []trace.Argument) error {
 	}
 
 	// Hashes
-	forkFeed.ParentHash = common.HashTaskID(uint32(forkFeed.ParentTid), forkFeed.ParentStartTime)
-	forkFeed.LeaderHash = common.HashTaskID(uint32(forkFeed.LeaderTid), forkFeed.LeaderStartTime)
-	forkFeed.ChildHash = common.HashTaskID(uint32(forkFeed.ChildTid), forkFeed.ChildStartTime)
+	forkFeed.ParentHash = hash.HashTaskID(uint32(forkFeed.ParentTid), forkFeed.ParentStartTime)
```

Comment:
What about making `hash.HashTaskID` more generic as `hash.HashID`? In the comments arg1 and arg2 should be better explained.

Reply:
I'm going to do another PR after this one where I want to check and stabilize the API of all of common packages